skfda/representation/_functional_data.py

```diff
@@ -85,6 +85,34 @@ def ndim_codomain(self):
         """
         return self.ndim_image
 
+    @abstractmethod
+    def image(self, dim=None):
```

Comment:
Maybe `coordinate_function` is too long, I think that `coordinate` may be better, what do you think? (the same with the parameter of `concatenate`, I prefer only `as_coordinate`)

Reply:
Seems ok. Another think that probably would improve this PR is that `coordinates` were an object with a `__getitem__` method, so that you can do `f.coordinates[2]` or `f.coordinates[3:8]` or `for g in f.coordinates`. This seems more pythonic than the alternative.

Reply:
Ok, I will change it

Reply:
Changed in b985c5d.